framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/tpch_sf100_parquet_explain.json

```diff
@@ -0,0 +1,21 @@
+{
+  "testId": "runtime_filter_tpch_sf100_parquet_plan",
+  "type": "group",
+  "description": "Plan Validation for Runtime Filter on TPC-H parquet data",
+  "categories": [
+    "advanced"
+  ],
+  "matrices": [
+    {
+      "query-file": ".*.q.explain",
```

Comment:
Can we use .sql extension? I'm trying to move all tests to use the .sql, .sql.explain, .plan & .e_tsv extensions.